readme.md

```diff
@@ -118,6 +118,7 @@ Click the type names for complete docs.
 - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
 - [`Simplify`](source/simplify.d.ts) - Useful to flatten the type output to improve type hints shown in editors. And also to transform an interface into a type to aide with assignability.
 - [`Get`](source/get.d.ts) - Get a deeply-nested property from an object using a key path, like [Lodash's `.get()`](https://lodash.com/docs/latest#get) function.
+- [`Schema`](source/schema.d.ts) - Recursively replaces all the property values into a given value type for an object.
```

Comment:
The description makes it sound like it mutates the existing type, but it creates a new type. See the description of the other "deep" types.